shapely/geometry/base.py

```diff
@@ -92,19 +92,18 @@ def __nonzero__(self):
         return self.__bool__()
 
     def __repr__(self):
-        class_name = str(self.__class__.__name__)
         try:
             wkt = super().__str__()
         except (GEOSException, ValueError):
             # we never want a repr() to fail; that can be very confusing
-            return "<shapely.{} Exception in WKT writer>".format(class_name)
+            return "<Exception in WKT writer>"
```

Comment:
Maybe in this specific case we could still include the class? (so you know whether it is a MultiPoint, or ...)